cleo_plugins/DebugUtils/ScreenLog.h

```diff
@@ -84,6 +84,11 @@ class ScreenLog
                             this->msg.push_back(c);
                     }
                 }
+
+                if(!this->msg.empty() && this->msg.back() == ' ') // a bug(?) in game prevents drawing texts ending with whitespace
+                {
+                    this->msg.back() = '_'; // '_' is drawn as empty character too
+                }
```

Comment:
maybe just trim trailing whitespace?

Reply:
There might be another one. And that's not what user wanted to print.